params/params.go

```diff
@@ -0,0 +1,26 @@
+package params
+
+import (
+	"github.com/juju/bundlechanges"
+	"github.com/juju/httprequest"
+)
+
+// ChangesResponse contains the results of parsing a bundle into a list of
+// changes.
+type ChangesResponse struct {
+	Changes []bundlechanges.Change
+}
+
+// ChangesRequest contains the bundle which is to be parsed into a list of
+// changes.
+type ChangesRequest struct {
+	Bundle string
```

Comment:
Perhaps this should be:

     Bundle string `json:"bundle"`

because I think people like lower-case JSON attributes.